nautobot/extras/utils.py

```diff
@@ -80,6 +81,40 @@ def get_choices(self):
         return [(f"{ct.app_label}.{ct.model}", ct.pk) for ct in ContentType.objects.filter(self.get_query())]
 
 
+@deconstructible
+class ModelSubclassesQuery:
```

Comment:
This is now specific to the `tags` field - can we rename it to something less generic, like `TaggableClassesQuery` or similar?